vars/getJenkinsLabels.groovy

```diff
@@ -17,6 +17,7 @@ def call(String backend, String aws_region=null) {
                           'gce': 'gce-sct-builders',
                           'k8s-gce-minikube': 'gce-sct-builders',
                           'k8s-gke': 'gce-sct-builders',
+                          'k8s-eks': 'aws-sct-builders-eu-west-1-new',
```

Comment:
remember to remove when moving to SCT Runner infra

Reply:
removed